pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -458,6 +461,27 @@ type SNIIngress struct {
 	Labels map[string]string `json:"labels,omitempty"`
 }
 
+// ExposureClassHandler contains configuration for an exposure class handler.
+type ExposureClassHandler struct {
+	// Name is the name of the exposure class handler.
+	Name string `json:"name"`
+	// LoadBalancerService contains configuration which is used to configure the underlying
+	// load balancer to apply the control plane endpoint exposure strategy.
+	LoadBalancerService LoadBalancerServiceConfig `json:"loadBalancerService"`
+	// SNI contains optional configuration for a dedicated ingressgateway belonging to
+	// an exposure class handler. This is only required in context of the APIServerSNI feature of the gardenlet.
+	// +optional
+	SNI *SNI `json:"sni,omitempty"`
+}
+
+// LoadBalancerService contains configuration which is used to configure the underlying
+// load balancer to apply the control plane endpoint exposure strategy.
+type LoadBalancerServiceConfig struct {
+	// Annotations is a key value map to annotate the underlying load balancer services.
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Why is it optional? Should the whole `LoadBalancerService` be optional then? 👀

Reply:
No, it shouldn't be optional. Will remove it plus the belonging validation.

Reply:
done